lte/gateway/python/magma/policydb/apn_rule_map_store.py

```diff
@@ -0,0 +1,40 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from lte.protos.policydb_pb2 import SubscriberPolicySet
+from magma.common.redis.client import get_default_client
+from magma.common.redis.containers import RedisHashDict
+from magma.common.redis.serializers import get_proto_deserializer, \
+    get_proto_serializer
+
+
+class ApnRuleAssignmentsDict(RedisHashDict):
```

Comment:
I wonder if it would make sense to just consolidate all the *_Dict classes into one file. They're all essentially doing the same things.